.claude/skills/deploy-test/SKILL.md

```
@@ -11,8 +11,8 @@ Deploy and test the Subnet Calculator on the dev server:
    rsync -a --delete Subnet-Calculator/ root@192.168.80.15:/opt/container_data/dev.seanmousseau.com/html/claude/subnet-calculator/
    scp testing/fixtures/iframe-test.html root@192.168.80.15:/opt/container_data/dev.seanmousseau.com/html/claude/subnet-calculator/
    ```
-2. Run the CDP browser test suite:
+2. Run the Playwright browser test suite:
    ```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)